crates/rpc/src/pathfinder/methods/get_proof.rs

```diff
@@ -16,6 +17,13 @@ pub struct GetProofInput {
     pub keys: Vec<StorageAddress>,
 }
 
+#[derive(Deserialize, Debug, PartialEq, Eq)]
+pub struct GetProofInputClass {
```

Comment:
```suggestion
pub struct GetClassProofInput {
```

Following the naming of the endpoint (`pathfinder_getClassProof`), this should be renamed `GetClassProofInput`.